dev.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM nginx:alpine
+
+RUN apk add perl perl-dev libxml2-utils make
+RUN cpan install -T Parse::RecDescent
```

Comment:
This can be `apk add perl-parse-recdescent`  in the previous RUN step

Reply:
Nice! Means we can skip `perl-dev`. I'll make the same change to the main **Dockerfile**.